superset/reports/commands/execute.py

```diff
@@ -181,7 +175,7 @@ def _get_url(
             user_friendly=user_friendly,
             dashboard_id_or_slug=self._report_schedule.dashboard_id,
             standalone=DashboardStandaloneMode.REPORT.value,
-            force=force,
+            # force=force,  TODO (betodealmeida) implement this
```

Comment:
This is currently not supported by the dashboard API (only the chart API), so I thought it would be better to leave it out for now.